ctms/acoustic_service.py

```diff
@@ -212,6 +213,11 @@ def _main_table_converter(self, contact):
                         acoustic_field_name
                         in AcousticResources.VALID_ACOUSTIC_MAIN_TABLE_FIELDS
                     ):
+                        if acoustic_field_name == "fxa_created_date":
+                            inner_value = self.fxa_created_date_string_to_datetime(
+                                inner_value
+                            )
+
```

Comment:
If fxa_created_date is a string it will be parsed into a datetime, if it was not an instanceof string then it's a no-op.

The inner_value is converted from datetime to acoustic_readable in the following few lines `self.transform_field_for_acoustic`